StripeUICore/StripeUICore/Source/Elements/DropdownFieldElement.swift

```diff
@@ -66,7 +80,7 @@ import UIKit
         if #available(macCatalyst 14.0, *) {
```

Comment:
Not sure why the diff shows red-out on macCatalyst and 14.0. I see it in my branch stil..